Externals.cfg

```diff
@@ -1,9 +1,9 @@
 [regional_workflow]
```

Comment:
Sorry for the late review here...please remove these Externals.cfg changes from your PR, these should not be committed to master